camp_fin/views.py

```diff
@@ -199,72 +239,13 @@ def get_context_data(self, **kwargs):
         context['pac_objects'] = pac_objects
         context['candidate_objects'] = candidate_objects
 
-        # Race for governor
```

Comment:
i wonder, if this something we will want again for the future?

Reply:
It's possible? The way @jeancochrane built that was so that they can more or less pick out which races to feature and it's up to us to implement (which I think is fine). Since there won't be another Gubernatorial election until 2022, I think we're probably safe removing it.

Reply:
hm, ok. it feels like a shame to strike this code if there's potential to need it again. are you opposed to implementing something like `if it's after the configured election year, show the lobbyists, else show the races`?

Reply:
thanks for opening #159! that'll do for me on this point.